proto/objectives/v1alpha1/objectives.proto

```diff
@@ -45,7 +45,8 @@ message Indicator {
   oneof options {
     Ratio ratio = 1;
     Latency latency = 2;
-    BoolGauge boolGauge = 3;
+    LatencyNative latency_native = 3;
+    BoolGauge boolGauge = 4;
```

Comment:
I'm 99% sure that this is fine because nobody is depending on this definition, but changing the message number is a breaking change isn't it?

Reply:
You're write. I shouldn't care about the order here and just add the latency_native as 4th. :+1: